buildlib/az-distro-release.yml

```diff
@@ -47,14 +47,9 @@ jobs:
       - bash: |
           set -eEx
           ./autogen.sh
-          mkdir pkg-build
+          mkdir -p pkg-build
```

Comment:
Seems -p is redundant here

Reply:
Might be superficial here, but does not harm.
I've seen at least one CI failure, where this command was executed before the parent folder was created.